models/commentsModels.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection.js");
+
+const fetchCommentsOfReview = (reviewId) => {
+  return db
+    .query(
+      `SELECT * FROM comments WHERE review_id = $1 ORDER BY created_at DESC`,
+      [reviewId]
+    )
+    .then((response) => {
+      const commentsOfReview = response.rows;
+      return commentsOfReview;
+    })
+    .catch((err) => {
```

Comment:
This catch block is doubling up on the catch block inside the ontroller. I would suggest it is unnecessary and should therefore be removed.